mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -172,3 +172,54 @@ def accept_or_reject(self):
             logger.exception(f'Check {self.object_id} could not be actioned')
             self.add_error(None, _('There was an error with your request.'))
             return False
+
+
+class AssignCheckToUserForm(GARequestErrorReportingMixin, forms.Form):
+    assignment = forms.CharField(max_length=10)
+
+    def __init__(self, object_id, request, **kwargs):
+        super().__init__(**kwargs)
+        self.object_id = object_id
+        self.request = request
+
+    @cached_property
+    def session(self):
+        return get_api_session(self.request)
+
+    def get_object(self):
+        """
+        Gets the security detail object, a sender or prisoner profile
```

Comment:
i think this was copied from somewhere else

Reply:
Ah yes, I think it's left from when the view was trying to render a template, which we aren't doing any more.